<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="start">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>400</y>
         <width>351</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Построить сигнал</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>701</width>
         <height>120</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>t(0;1)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QRadioButton" name="sig_triug">
          <property name="text">
           <string>Триугольный импульс</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QRadioButton" name="sig_garm">
          <property name="text">
           <string>Гармонический сигнал</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>       phi</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>       f0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" colspan="2">
         <widget class="QLineEdit" name="lineEdit_7">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>       f0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" colspan="2">
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="10">
         <widget class="QLineEdit" name="lineEdit_6">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QRadioButton" name="sig_lchm">
          <property name="text">
           <string>ЛЧМ</string>
          </property>
         </widget>
        </item>
        <item row="1" column="8">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>        Fd</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" colspan="2">
         <widget class="QLineEdit" name="lineEdit_1">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>        Fd</string>
          </property>
         </widget>
        </item>
        <item row="1" column="6" colspan="2">
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string> delta f</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6" colspan="2">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="10">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QCustomPlot" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>701</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>766</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
